src/views/Deposit/components/InfoCard/components/SmallInfoCard.tsx

```diff
@@ -23,7 +23,33 @@ export const SmallInfoCard: React.FC<ISmallInfoCard> = ({ statContent, statFill,
     <StyledSmallInfoCard>
       <StyledSmallInfoCardHeader>
         <StyledStatText color={colors.white}>{statTitle}</StyledStatText>
-        {statType === 'question' ? <QuestionMarkSvg /> : <MultiplierSvg />}
+        {statType === 'question' ? (
+          <button
+            css={`
+              background: transparent;
+              border: none;
+
+              &:hover {
+                cursor: pointer;
+              }
+            `}
+          >
+            <QuestionMarkSvg />
+          </button>
+        ) : (
+          <button
+            css={`
+              background: transparent;
+              border: none;
+
+              &:hover {
+                cursor: pointer;
+              }
+            `}
+          >
+            <MultiplierSvg />
+          </button>
```

Comment:
This button `css` is repeated, so we should probably make it a component :)

Reply:
Fixed in c1035d70b1e2f4659a79cba790c85b905ed8d5f2